docs/how-to-guides/deploy-a-flow/deploy-using-dev-server.md

```diff
@@ -14,6 +14,11 @@ an example to show how to deploy a flow.
 Please ensure you have [create the connection](../manage-connections.md#create-a-connection) required by flow, if not, you could
 refer to [Setup connection for web-classification](https://github.com/microsoft/promptflow/tree/main/examples/flows/standard/web-classification).
 
+Note: We will use relevant environment variable to override connection configurations in serving mode. For instance, 
```

Comment:
We will use relevant environment variable ({connection_name}_{key_name}) to override connection configurations in serving mode, white space in connection name will be removed directly from environment variable name.

Reply:
updated